db/init/100_views.sql

```diff
@@ -114,6 +136,12 @@ LEFT JOIN LATERAL (
   WHERE threads.id = comments.parent_thread) comments ON TRUE
 );
 
+/**
+ * This view returns details about thread details as they appear to a user.
```

Comment:
I feel like there's probably a better way to phrase this, it tripped me up when I initially read it. Something like `returns information about thread details` ? `returns specifics about thread details` ? Would just `returns thread details as they appear to a user` be an accurate descriptor on its own?